test/test_serde_full.py

```diff
@@ -1318,7 +1328,7 @@ def compare(detailed, original):
 # syft.frameworks.torch.tensors.interpreters.promise.PromiseTensor
 def make_promisetensor(**kwargs):
     pt = syft.Promise.FloatTensor(shape=torch.Size((3, 4)))
-    pt.tags = ["tag1"]
+    pt.tag("tag1")
     pt.description = "I promise"
```

Comment:
Q: Shouldn't we do the same for ```description```? Maybe ```pt.describe("I promise")```.

Also, when is ok to use ```tag``` vs ```tags```?
Should ```tags``` be used for TorchTensors (since we know we are working with tensors) and ```tag``` for ```AbstractObject``` (and we do not know the underlying object?)?

Reply:
Thanks! I've updated to `.describe()` everywhere.
I think I shouldn't have used `.tags` and `.description` properties directly at all. Just didn't know about them 🤷‍♂